spec/models/manageiq/providers/microsoft/infra_manager/provision_workflow_spec.rb

```diff
@@ -16,6 +16,27 @@
     described_class.new({}, admin.userid)
   end
 
+  describe "#load_hosts_vlans" do
+    let(:host)        { FactoryGirl.create(:host_with_ref) }
+    let(:switch)      { FactoryGirl.create(:switch) }
+    let(:lan_parent)  { FactoryGirl.create(:lan) }
+    let(:lan)         { FactoryGirl.create(:lan) }
```

Comment:
So you can definitely do `FactoryGirl.create(:lan, :parent => lan_parent, :switch => switch)` and avoid the `lan.update_attributes`